src/mgr/QueryParams.cc

```diff
@@ -65,61 +69,69 @@ Mgr::QueryParams::find(const String& name) const
     return iter;
 }
 
-bool
-Mgr::QueryParams::ParseParam(const String& paramStr, Param& param)
+/// Parses the value part of a "param=value" URL section.
+/// Value can be a comma separated list of integers, or an opaque string.
+/// \note opaque string can be a list with any non-integer.
+///       For example; 1,2,3,4,5,z
```

Comment:
I used the English formal form here for the note to explicitly clarify which of the text is the example and which is the "for example". The abbreviated form has a visual ambiguity about whether everything inside the brackets is the example string.

Reply:
> I used the English formal form here

FWIW, I have not seen the form you used (i.e. `For Example; ...`) used in this corner of the world. I could not find any examples of it on the web either, but it is difficult to google punctuation, of course. That form also looks very odd to me because semicolons usually separate independent thoughts/parts from each other rather than delimit labels from the information they label or introductory phrases from the primary ones.

> The abbreviated form has a visual ambiguity about whether everything inside the brackets is the example string.

I do not see any ambiguity -- the suggested parens are a very common format for inlined examples and "that is" clarifications -- but you can surround `1,2,z` with quotes if you think it will help more than it will hurt (i.e. `"1,2,z"`).

Reply:
Done in 239abba